src/SqlLocalDb/MartinCostello.SqlLocalDb.csproj

```diff
@@ -46,9 +46,9 @@
     <PackageReference Update="Microsoft.Extensions.Logging.Abstractions" VersionOverride="6.0.0" />
   </ItemGroup>
   <ItemGroup Condition="$([MSBuild]::IsTargetFrameworkCompatible('$(TargetFramework)', 'net8.0'))">
-    <PackageReference Update="Microsoft.Data.SqlClient" VersionOverride="5.1.5" />
-    <PackageReference Update="Microsoft.Extensions.DependencyInjection.Abstractions" VersionOverride="8.0.0" />
-    <PackageReference Update="Microsoft.Extensions.Logging.Abstractions" VersionOverride="8.0.0" />
+    <PackageReference Update="Microsoft.Data.SqlClient" VersionOverride="5.2.1" />
+    <PackageReference Update="Microsoft.Extensions.DependencyInjection.Abstractions" VersionOverride="8.0.1" />
+    <PackageReference Update="Microsoft.Extensions.Logging.Abstractions" VersionOverride="8.0.1" />
```

Comment:
These versions are intentionally pinned to the version that was chosen at the point support for .NET 8 was first added.

Why do you feel these need to be updated? Consuming applications can update their own transient dependencies to pick up fixes for any vulnerabilities.